lib/big_query/base/timestamp_group.ex

```diff
@@ -0,0 +1,48 @@
+defmodule BigQuery.Base.TimestampGroup do
+  import BigQuery.Base.Query
+  import BigQuery.Base.Timestamp,
+    only: [timestamp_intervals: 3, timestamp_params: 2]
+
+  def group_query(tbl, where_sql, params, interval, grouping) do
+    params
+    |> timestamp_params(interval)
+    |> group_params(grouping)
+    |> query(group_sql(tbl, where_sql, grouping))
+  end
+
+  def group_sql(tbl, where_sql, grouping) do
+    """
+    WITH
+      intervals AS (#{timestamp_intervals(tbl, where_sql, grouping.fkey)}),
+      top_groups AS (#{top_groups("intervals", grouping.fkey)})
+    SELECT
+      time,
+      SUM(count) as count,
+      IF(rank IS NULL, NULL, ANY_VALUE(#{grouping.display})) AS display,
```

Comment:
haven't SQLed in a while-- so this is saying if rank is null, then set display to null too, else use the country_name value as display? why use the ANY_VALUE thing?